apps/meteor/ee/server/lib/engagementDashboard/channels.ts

```diff
@@ -40,6 +40,7 @@ export const findAllChannelsWithNumberOfMessages = async ({
 		end: convertDateToInt(end),
 		startOfLastWeek: convertDateToInt(startOfLastWeek),
 		endOfLastWeek: convertDateToInt(endOfLastWeek),
+		onlyCount: false,
```

Comment:
Maybe this should be optional? (Options should be optional too)

Reply:
I suggested @totoi690 to split it in two different methods: one for retrieving the results, and other one only for counting. Please check this later